control/tap-ctl-spawn.c

```diff
@@ -172,8 +174,42 @@ tap_ctl_get_child_id(int readfd)
 	return id;
 }
 
+/* Move the process to the nominated cgroup slice */
+static int
```

Comment:
This function returns values that are never used.
I don't think we are ever using errno in the calling function to do anything meaningful, so
I'd suggest to just return -1 (errno value is already logged in EPRINTF) and let the caller decide
what to do.
See my comments when this function is called.